apps/frontend/app/root.tsx

```diff
@@ -113,7 +127,11 @@ const DefaultHeadTags = () => {
 				content="minimum-scale=1, initial-scale=1, width=device-width, shrink-to-fit=no, user-scalable=no, viewport-fit=cover"
 			/>
 			<link rel="manifest" href="/manifest.json" />
-			<link rel="apple-touch-icon" href="/apple-touch-icon.png" />
+			{isIOS18 ? (
```

Comment:
Instead of conditionally rendering the whole element, please put the ternary inside the `href` prop.